package.json

```diff
@@ -35,13 +35,21 @@
   },
   "dependencies": {
     "browserify": "^12.0.1",
+    "express": "https://registry.npmjs.org/express/-/express-4.14.0.tgz",
```

Comment:
hmm that's very strange I just did the usual `npm i express -save`

I'll sort it out